metal-stack-cloud/ansible-roles
# Use meilisearch from mini-lab #34

This change is split over the following PRs to implement https://github.com/metal-stack-cloud/deployment/issues/34:

- https://github.com/metal-stack-cloud/ansible-roles/pull/11
- https://github.com/metal-stack-cloud/deployment/pull/35

It removes the meilisearch ansible role from [metal-stack-cloud/ansible-roles](https://github.com/metal-stack-cloud/ansible-roles) and replaces it with the already existing [metal-stack/mini-lab](https://github.com/metal-stack/mini-lab).

metal-stack will use the `metal-` prefix, while the api-server of metal-stack-cloud uses the `cloud-` prefix in meilisearch.

**How to test?**

- checkout deployment `feature/use-metal-auditing` [metal-stack-cloud/deployment#35](https://github.com/metal-stack-cloud/deployment/pull/35)
- either link the ansible-roles in `deployment/docker-compose.yml` at branch `feature/use-metal-auditing`...
- ...or in `deployment/requirements.yaml` change the version from `main` to `feature/use-metal-auditing`
- `make up`
- visit http://auditing.172.17.0.1.nip.io:8080/
- enter the secret master key `geheim` (only for debugging!)
- you should see at least two indexes
- one prefixed with `metal-`
- one prefixed with `cloud-`
- the more you do in the console or by using the metalctl, the more entries you should see

![image](https://user-images.githubusercontent.com/2100336/218490929-d6260020-ba8a-45cd-8886-1f13cbb3fec9.png)
